specification/signalr/resource-manager/Microsoft.SignalRService/preview/2021-04-01-preview/signalr.json

```diff
@@ -0,0 +1,2224 @@
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "SignalRSharedPrivateLinkResources_List": {
+            "$ref": "./examples/SignalRSharedPrivateLinkResources_List.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.SignalRService/signalR/{resourceName}/sharedPrivateLinkResources/{sharedPrivateLinkResourceName}": {
+      "get": {
+        "tags": [
+          "SignalR"
+        ],
+        "description": "Get the specified shared private link resource",
+        "operationId": "SignalRSharedPrivateLinkResources_Get",
+        "parameters": [
+          {
+            "name": "sharedPrivateLinkResourceName",
+            "in": "path",
+            "description": "The name of the shared private link resource",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success. The response describes a shared private link resource.",
+            "schema": {
+              "$ref": "#/definitions/SharedPrivateLinkResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "SignalRSharedPrivateLinkResources_Get": {
+            "$ref": "./examples/SignalRSharedPrivateLinkResources_Get.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "SignalR"
+        ],
+        "description": "Create or update a shared private link resource",
+        "operationId": "SignalRSharedPrivateLinkResources_CreateOrUpdate",
+        "parameters": [
+          {
+            "name": "sharedPrivateLinkResourceName",
+            "in": "path",
+            "description": "The name of the shared private link resource",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "The shared private link resource",
+            "required": false,
```

Comment:
>"required": false, [](start = 12, length = 18)

Is this ok? Can you invoke a PUT call without a body?

Reply:
Nice catch, we didn't pay attention to it before. Fixed.